code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -97,17 +97,17 @@
 
 /obj/item/reagent_containers/hypospray/medipen
 	name = "epinephrine medipen"
-	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge."
+	desc = "A rapid and safe way to stabilize patients in critical condition for personnel without advanced medical knowledge. Contains a powerful preservative that can delay decomposition when applied to a dead body."
 	icon_state = "medipen"
 	item_state = "medipen"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
-	amount_per_transfer_from_this = 10
-	volume = 10
+	amount_per_transfer_from_this = 13
+	volume = 13
 	ignore_flags = 1 //so you can medipen through hardsuits
 	reagent_flags = DRAWABLE
 	flags_1 = null
-	list_reagents = list(/datum/reagent/medicine/epinephrine = 10)
+	list_reagents = list(/datum/reagent/medicine/epinephrine = 10, /datum/reagent/toxin/formaldehyde = 3)

```

Comment:
This metabolizes into histamine, no bueno

Reply:
less than 15% chance of some blurry vision and itching, nothing you should care about while being revived or crawling around in crit.

3u formaldehyde is not dangerous.

Reply:
histamine is a trash toxin anyways unless you OD on it, which requires you to hit 2-6 iterations from 15 iterations /w 5% chance per epipen. It's selling point is the low metab rate and the fact it lingers in the body until it does damage, but items like multiver and calo purge at a flat amount so 🤷‍♂ .

Not that anyone makes it, but there's also a couple of meds dedicated to purging it that would consequently also benefit multiver.

Reply:
> histamine is a trash toxin anyways

This is approximately what you said about multiver.

Reply:
are you implying something or am I suppose to read between the lines here?

Multiver was/is a bad toxin, the catch being you (the victim) need to know how to respond to it that isn't exclusively just pumping yourself with well... multiver. Not sure why that's relevant here though.